assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -1,5 +1,5 @@
 import Component from "@ember/component";
-import { clipboardCopy } from "discourse/lib/utilities";
+import { clipboardCopyAsync } from "discourse/lib/utilities";
```

Comment:
Maybe we should add some discourse compatibility, so the plugin will not crash on older versions of Discourse. Something like https://github.com/discourse/discourse-assign/blob/main/.discourse-compatibility